ext/yahttp/yahttp/reqresp.cpp

```diff
@@ -2,6 +2,8 @@
 
 namespace YaHTTP {
 
+  template class AsyncLoader<Request>;
```

Comment:
Unrelated change

Reply:
That's the backport of #8640 mentioned in the description. I can make that into a separate backport PR but then I can't test that one on my system :)

Reply:
OK, right.